int64 numberExperiments
float64 positiveFeedbackPercentage
